frontend/src/reducers/routesReducer.js

```diff
@@ -1,14 +1,49 @@
 const initialState = {
     favouriteRoutes: [],
-    createdRoutes: []
+    createdRoutes: [{
+        routetitle: "abc",
+        routedescription: "abc",
+        routedistance: "abc",
+        routetime: "abc",
+        lat: 0,
+        long: 0,
+        route: [
+            [
+                [51.5, -0.1],
+                [51.5, -0.12],
+                [51.52, -0.12],
+            ],
+            [
+                [51.5, -0.05],
+                [51.5, -0.06],
+                [51.52, -0.06],
+            ],
+        ]
+    },
+        {
+            routetitle: "ROUTE 2",
+            routedescription: "THIS IS SOMEWHERE ELSE",
+            routedistance: "abc",
+            routetime: "abc",
+            lat: 0,
+            long: 0,
+            route: [
+                [
+                    [53.5, -0.1],
+                    [53.5, -0.12],
+                    [53.52, -0.12],
+                ],
+
+            ]
+        }]
```

Comment:
> Are these just placeholder for testing?

It might be good to have these dummy data there for now:
- It's easier to make sure things are functioning when the associated backend is not yet set up
- An easy reference of the data structure (routedistance should be ignored tho) when developing other route related frontend tasks

We can certainly remove it once we have test data in the backend.

Reply:
Okay, sounds good! I'll accept the changes.